app/styles/partials/navbar.scss

```diff
@@ -1,8 +0,0 @@
-@media (max-width: 767px) {
```

Comment:
Use semantic-ui's own responsive helpers and the custom helpers already added to the project